studunt.sql

```diff
@@ -0,0 +1,39 @@
+	/1/
+create table Student(
+
+STU-NUM CHAR(6) primary key,

```

Comment:
This is not the proper syntax for primary key Garam. should be   primary key (STU_NUM),